cli/test/helpers/onboarding-fixtures.mjs

```
@@ -0,0 +1,124 @@
+//   • the iOS error screen (ish.ErrorStep) — its recovery advice is unbounded
+//     (42–54 rows), so it routes through the same fullscreen scroll viewer as
+//     the AI analysis; only its COMPACT form renders inline, and that's ~20 rows.
+//
+// Each entry: { label, el, withProgress, platform }. withProgress=false for the
+// takeover / pre-flow frames that hide the progress bar (welcome, no-platform,
+// build-complete, adding-platform).
+import React from 'react'
+import * as ks from '../../src/build/onboarding/ui/steps/android-keystore.tsx'
+import * as sa from '../../src/build/onboarding/ui/steps/android-sa-gcp.tsx'
+import * as ash from '../../src/build/onboarding/ui/steps/android-shared.tsx'
+import * as aci from '../../src/build/onboarding/ui/steps/android-ci.tsx'
```

ESLint

[error] Expected "../../src/build/onboarding/ui/steps/android-ci.tsx" to come before "../../src/build/onboarding/ui/steps/android-shared.tsx".

(perfectionist/sort-imports)